src/main/java/com/flansmod/common/driveables/EntitySeat.java

```diff
@@ -817,6 +815,12 @@ public EntitySeat getSeat(EntityLivingBase living)
 	{
 		return this;
 	}
+  
+	// Check and update whether currret seat is driver seat
```

Comment:
Remove this comment please, it's not needed.

Reply:
deleted